AsyncDisplayKit/ASNetworkImageNode.mm

```diff
@@ -169,6 +171,19 @@ - (void)setDelegate:(id<ASNetworkImageNodeDelegate>)delegate
 - (void)displayWillStart
 {
   [super displayWillStart];
+  
+  if (_cacheSupportsSynchronousFetch) {
+    {
```

Comment:
Don't need the additional lock scope - can contain it with the if.  I think it would be slightly more readable.